Course.php

```diff
@@ -103,11 +103,11 @@
  
 $resultx1 = mysqli_query($con,$sql);
     
-while($row = mysqli_fetch_assoc($resultx1)) {$group_id=$row['Course_Group_id'];}  
+while($row = mysqli_fetch_assoc($resultx1)) {$_SESSION['group_id']=$row['Course_Group_id'];}  
```

Comment:
- What is $resultx1?

- `$_SESSION['group_id']=$row['Course_Group_id'];`  There should be two spaces around the assignment operator (=).  Why? Easier for eyes.

**Hui**

Reply:
$resultx1 its a variable name.
Will take note on the spaces. Actually i think we might need a code cleaner willing to go through the entire script and cleanup unnecessary duplicate, spaces, spelling errors etc and for all new lines and adjustments we will make sure to comment them and make them neat.

Reply:
> $resultx1 its a variable name.
> Will take note on the spaces. Actually i think we might need a code cleaner willing to go through the entire script and cleanup unnecessary duplicate, spaces, spelling errors etc and for all new lines and adjustments we will make sure to comment them and make them neat.

@Teecloudy 

Code cleaning is long waited.  Thanks for offering to do that.

Hui

Reply:
hahah i wasn't really referring to myself to do the cleaning but if there will be no one to do it, i will go on with it. I'm sure it will benefit future editors.

**Ashly**